File: Easy/Addem-up/Addem-up.swift
let N = Int(readLine()!)!
var cards : [Int] = readLine()!.split(separator: " ").map{Int($0)!}

var cost = 0
while cards.count > 1 {
    cards.sort()
    let num = cards.remove(at: 1) + cards.remove(at: 0)
    cost += num
    cards.append(num)
}
print(cost)

File: Easy/Addem-up/Addem-up.java
class Solution {
    public static void main(String args[]) {
        var in = new java.util.Scanner(System.in);
        int N = in.nextInt(), cost = 0;
        var cards = new java.util.ArrayList<Integer>(N);
        for (int i = 0; i < N; i++)
            cards.add(in.nextInt());
        for (int num; cards.size() > 1; cards.add(num)) {
            cards.sort(null);
            cost += (num = cards.remove(1) + cards.remove(0));
        }
        System.out.println(cost);
    }
}
